src/api/ddoc/views.rst

```diff
@@ -157,14 +157,13 @@
     transfer size for attachments.
 
 .. http:post:: /{db}/_design/{ddoc}/_view/{view}
-    :synopsis: Returns certain rows for the specified stored view
+    :synopsis: Returns results for the specified stored view
```

Comment:
Very minor:  Are there unstored views?  I think this could be shortened to just

```suggestion
    :synopsis: Returns results for the specified view
```

Reply:
Good call! I'll commit your suggestion.